src/core/system_parameters/gnss_satellite.cc

```diff
@@ -183,7 +182,7 @@ void Gnss_Satellite::set_PRN(uint32_t PRN_)
         }
     if (system == "GPS")
         {
-            if (PRN_ < 1 or PRN_ > 32)
+            if (PRN_ < 1 or (PRN_ > 32 and PRN_ < 120))
```

Comment:
Consider writing your 3-way compares consistently with those languages that support that expression
```suggestion
            if (PRN_ < 1 or (32 <  PRN_ and PRN_ < 120))
```

Reply:
While you described your overloading of GPS (and I don't think you originated it) for SBAS, this is a place where it would be worth making the distinction.

BY DEFINITION, the GPS constellation will not be more than 32 satellites. The higher numbered satellites are not (really) GPS satellites, are they? So is this a valid GPS error bound anymore?